src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -204,8 +204,12 @@ message PageStreamingConfigProto {
 // `PageStreamingRequestProto` defines which fields match the paging pattern in
 // the request.
 message PageStreamingRequestProto {
```

Comment:
Out of scope for this CL, but "streaming" confuses me because there are no streaming RPCs involved.